src/components/Frame/components/Loading/Loading.tsx

```diff
@@ -1,79 +1,41 @@
-import React, {Component} from 'react';
-import debounce from 'lodash/debounce';
+import React, {useEffect, useState} from 'react';
 
-import styles from './Loading.scss';
-
-export interface LoadingProps {}
+import {useI18n} from '../../../../utilities/i18n';
 
-interface State {
-  progress: number;
-  step: number;
-  animation: number | null;
-}
+import styles from './Loading.scss';
 
 const INITIAL_STEP = 10;
 const STUCK_THRESHOLD = 99;
 
-export class Loading extends Component<LoadingProps, State> {
-  state: State = {
-    progress: 0,
-    step: INITIAL_STEP,
-    animation: null,
-  };
-
-  private ariaValuenow = debounce(() => {
-    const {progress} = this.state;
-    return Math.floor(progress / 10) * 10;
-  }, 15);
```

Comment:
Removed all this sillyness for a [CSS transition delay](https://github.com/Shopify/polaris-react/pull/3770/files#diff-5cd4dd79a5c5206628e3b059fadc942878686fcb3830fd86d2de08d8de774661R17).